src/abstract/lib/types/multiInformant.ts

```diff
@@ -8,4 +8,8 @@ export type MultiInformantState = {
   currentUserSubject?: MultiInformantSubject;
   sourceSubject?: MultiInformantSubject;
   targetSubject?: MultiInformantSubject;
+  appletId?: string;
```

Comment:
Even though there are other ways to access current `appletId`, it was convenient to just include it here for consumption by `TakeNowSuccessModal`, which derives all its needed props from the most recent `MultiInformantState` stored in Redux.